FlowCrypt/Controllers/Setup/SetupCreatePassphraseViewController.swift

```diff
@@ -188,6 +195,48 @@ extension SetupGenerateKeyViewController {
         }
     }
 
+    private func setupAccountWithExistingKeys(with passPhrase: String) {
```

Comment:
This, once moved into `SetupWithKeysFromEkmViewController` should be renamed to `setupAccountWithKeysFetchedFromEkm`